### PR TITLE
CES-6339 sync fix

### PR DESCRIPTION
**Pull request by mboudreau**
**What's in this PR?**
Fixing things preventing sync from happening within a client.

**Why**
Cryptor failures would stop sync in it's track since the introduction of it to be used with DB secrets without reliable checks or retries

**Added feature flags**
None

**Affected issues**  
CES-6339

**How has this been tested?**  
Included unit tests are still passing

**Whats Next?**
Retry sync for customer and see if it passes.

### CODE REVIEW

File: src/routes/healthcheck/healthcheck-get.ts [hunk 1]
```diff
@@ -1,19 +1,16 @@
 import { Request, Response } from "express";
 import { getLogger } from "config/logger";
-import { envVars } from "config/env";
 import { deepCheckCryptor } from "./deep-check-cryptor";
 
 const logger = getLogger("healthcheck");
 export const HealthcheckGet = async (_: Request, res: Response): Promise<void> => {
 
-	if (envVars.MICROS_GROUP === "Worker") {
```
Comment (gxueatlassian) [inline src/routes/healthcheck/healthcheck-get.ts:9]: Webserver healtcheck will be hit very often, like every minute. Have you considered this?

File: src/transforms/transform-branch.ts [hunk 2]
```diff
@@ -36,22 +36,26 @@ export const transformBranch = async (github: GitHubInstallationClient, webhookP
...
-				id: getJiraId(ref),
-				issueKeys,
-				name: ref,
-				url: `${repository.html_url}/tree/${ref}`,
-				updateSequenceId: Date.now()
-			}
-		],
-		updateSequenceId: Date.now()
-	};
+	try {
```
Comment (gxueatlassian) [inline src/transforms/transform-branch.ts:39]: Can you please explain how is try/catch impacting the result?

File: src/routes/healthcheck/healthcheck-get.ts (hunk 1 above)
Comment (mboudreau) [reply src/routes/healthcheck/healthcheck-get.ts:9]: Are you suggesting that the cryptor sidecar can't handle 2 requests per minute?

Comment (gxueatlassian) [reply src/routes/healthcheck/healthcheck-get.ts:9]: It should be able to handle that, all good.

File: src/transforms/transform-branch.ts (hunk 2 above)
Comment (mboudreau) [reply src/transforms/transform-branch.ts:45]: I've added comments, logs and a better flow

**Review by gxueatlassian — APPROVED** [review]

File: src/worker/startup.ts [hunk 3]
```diff
@@ -12,6 +12,8 @@ export async function start() {
 		return;
 	}
 
+	await
```
Comment (krazziekay) [inline src/worker/startup.ts:15]: Whats this `await` for? Is this something that got missed out?

Comment (krazziekay) [reply src/worker/startup.ts:15]: I think this is causing the 500 Internal server error in the test case. Can you please revert this part?

Comment (mboudreau) [reply src/worker/startup.ts:15]: woops, good catch, it was something I was trying out.

**Review by gxueatlassian — APPROVED** [review]

**Review by krazziekay — APPROVED** [review]

**Pull request merged**